lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -184,6 +260,17 @@ Ufe::UndoableCommand::Ptr UsdContextOps::doOpCmd(const ItemPath& itemPath)
         return visibility->setCmd(
             current == UsdGeomTokens->invisible ? UsdGeomTokens->inherited : UsdGeomTokens->invisible);
     } // Visibility
+    else if (!itemPath.empty() && (itemPath[0] == "Add New Prim")) {
```

Comment:
Super minor comment: it may be a good idea to keep these operation strings (like "Add New Prim") in a unique place. That way it'd be easier to change if we ever need to. I've seen the "Add New Prim" string already three times during this review.

Reply:
Yes good idea. See commit [14d8d4b](https://github.com/Autodesk/maya-usd/pull/489/commits/14d8d4ba1fb39d2bb690ea1d3306b28964551749)